workflows/mcmicro.nf

```diff
@@ -39,141 +41,149 @@ workflow MCMICRO {
     ch_versions = Channel.empty()
     ch_multiqc_files = Channel.empty()
 
-    metadata = UPDATE_FROM_OME(ch_samplesheet, ch_markersheet)
-    ch_versions = ch_versions.mix(metadata.versions)
-
-    ch_samplesheet = metadata.samplesheet
-    ch_markersheet = metadata.markersheet
-
-    ch_samplesheet.dump(tag: "ch_samplesheet")
-    ch_markersheet.dump(tag: "ch_markersheet")
+    ch_samplesheet.map{meta, image_tiles, dfp, ffp -> [meta, image_tiles]} | BFTOOLS_SHOWINF
+    BFTOOLS_SHOWINF.out.xml.map{ meta, xml_path -> xml_path }.set{xml}
```

Comment:
Maybe see if we can also remove the call to `BFTOOLS_SHOWINF` from `UPDATE_FROM_OME`.